src/CodeGen_X86.cpp

```diff
@@ -33,7 +33,8 @@ Target complete_x86_target(Target t) {
     }
     if (t.has_feature(Target::AVX512_Cannonlake) ||
         t.has_feature(Target::AVX512_Skylake) ||
-        t.has_feature(Target::AVX512_KNL)) {
+        t.has_feature(Target::AVX512_KNL) ||
+        t.has_feature(Target::Znver3)) {
```

Comment:
Should it also enable other features that would have been autodetected for `znver3` otherwise?
(currently, `f16c` and `fma`)